CONTRIBUTING.md

```diff
@@ -521,16 +521,23 @@ can be used in these cases.
 ### Feature Flags
 
 Sometimes we want to introduce new breaking behavior because we believe this is
-the correct default behavior for the CDK. The problem of course is that breaking
-changes are only allowed in major versions and those are rare.
+the correct default behavior for the CDK. However, breaking changes are only allowed 
+in major versions and those are rare.
 
 To address this need, we have a feature flags pattern/mechanism. It allows us to
 introduce new breaking behavior which is disabled by default (so existing
 projects will not be affected) but enabled automatically for new projects
 created through `cdk init`.
 
+**DISCLAIMER**: feature flags are a "last resort" mechanism and should only be used
+when it is impossible to figure out a way to introduce a feature without breaking
+existing users. If you think your feature should be implemented behind a feature
+flag, you will need to get an approval from at least two core team members. We do that
+in order to avoid the abuse of this powerful capability.
+
 The pattern is simple:
 
+1. Seek the approval of a core team member that a feature flag can be used.
```

Comment:
list numbering is off, two "1"s.

Reply:
Fixed

Reply:
@MrArnoldPalmer - Technically, markdown allows numbered lists to contain any number for list items - https://daringfireball.net/projects/markdown/syntax#list. The rendered version will render this correctly.

I use `1.` everywhere as well so I don't have to re-number when I change the order or introduce a new item to the list.